src/ocm/components/clusterConfiguration/platformIntegration/PlatformIntegration.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { OcmSwitchField } from '../../ui/OcmFormFields';
+import {
+  isClusterPlatformTypeVM,
+  Cluster,
+  PopoverIcon,
+  PlatformType,
+  VSPHERE_CONFIG_LINK,
+} from '../../../../common';
+import useClusterSupportedPlatforms, {
+  SupportedPlatformIntegrationType,
+} from '../../../hooks/useClusterSupportedPlatforms';
+import PlatformIntegrationVsphere from './PlatformIntegrationVsphere';
+import { Link } from 'react-router-dom';
+
+type KeyType = Record<SupportedPlatformIntegrationType, string>;
+
+const undeterminedPlatformMessage =
+  'Enable platform integration to access features directly in OpenShift, like vSphere node auto-scaling and persistent storage or Nutanix node auto-scaling.';
+const vspherePlatformMessage =
+  "Enable vSphere platform integration to access features directly inside OpenShift, like node auto-scaling and persistent storage. You'll need to set vSphere configuration after cluster installation is complete.";
+{
```

Comment:
The few lines after the message are probably some kind of merge conflict?
Lines 23 to 26.